src/main/java/org/ossgang/commons/observable/DispatchingObservableValue.java

```diff
@@ -48,7 +46,7 @@ protected DispatchingObservableValue(T initial) {
     @Override
     public Subscription subscribe(Observer<? super T> listener, SubscriptionOption... options) {
         Set<SubscriptionOption> optionSet = new HashSet<>(Arrays.asList(options));
-        if (optionSet.contains(FIRST_UPDATE)) {
+        if (optionSet.contains(ObservableValueSubscriptionOption.FIRST_UPDATE)) {
```

Comment:
Did you remove these static imports on purpose, or was it IntelliJ?

Reply:
Intellij.. I just did refactor :confused: